test/offline/test_cql.py

```diff
@@ -227,3 +256,5 @@ def watch():
 
 if __name__ == "__main__":
     test_cql()
+    # test calibrated cql
+    test_cql(calibrated=True)
```

Comment:
Does it really test that the new functionality is performing what it's supposed to do?